examples/meta/cpp/CMakeLists.txt

```diff
@@ -45,3 +53,10 @@ ENDFOREACH()
 add_custom_target(build_cpp_meta_examples ALL
     DEPENDS ${GENERATED_CPP_EXAMPLES}
     COMMENT "Compiled generated cpp examples")
+
+IF (NOT ENABLE_TESTING)
+	MESSAGE(
```

Comment:
maybe you can put that to the place where you set the flags rather than down here?
Also, the indentation is a bit weird